packages/foreman/foreman/foreman.spec

```diff
@@ -270,6 +270,8 @@ BuildRequires: %{?scl_prefix}npm(compression-webpack-plugin) >= 1.1.11
 BuildRequires: %{?scl_prefix}npm(compression-webpack-plugin) < 1.2.0
 BuildRequires: %{?scl_prefix}npm(css-loader) >= 0.23.1
 BuildRequires: %{?scl_prefix}npm(css-loader) < 1.0.0
+BuildRequires: %{?scl_prefix}npm(cssnano) >= 3.4.0
+BuildRequires: %{?scl_prefix}npm(cssnano) < 4.1.1
```

Comment:
This doesn't match the version in core: https://github.com/theforeman/foreman/blob/develop/package.json#L43
Also, it's missing from the `Requires` section below